VisualPinball.Unity/VisualPinball.Unity.Editor/Inspectors/GateInspector.cs

```diff
@@ -78,5 +105,14 @@ public override void OnInspectorGUI()
 
 			base.OnInspectorGUI();
 		}
+
+		protected override void FinishEdit(string label, bool dirtyMesh = true)
+		{
+			if (label == "Two Way") {

```

Comment:
Not optimal since we might rename that label and it will break the repaint, but not sure about an alternative.

Reply:
I could put a static readonly and use it on both places

Reply:
Or a third optional parameter with an enum representing the field? It would only have two values (`Any`, `IsTwoWay`) for now. You can pass it to `ItemInspector.ItemDataField()` and receive it back in `FinishEdit()`.

Reply:
A lot of refactor if i change the FinishEdit prototype, maybe in another issue about fields edition ?

Reply:
An optional argument would add two lines of code (one to `ItemDataField(string label, ref bool field, bool dirtyMesh = true)`, and one to `FinishEdit(string label, bool dirtyMesh = true)`), no?

Reply:
Plus the enum definition of course.